ur5e_moveit_config/package.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+
+  <name>ur5e_moveit_config</name>
+  <version>0.0.0</version>
+  <description>
+     An automatically generated package with all the configuration and launch files for using the ur5 with the MoveIt Motion Planning Framework
+  </description>
+  <maintainer email="g.a.vanderhoorn@tudelft.nl">G.A. vd. Hoorn</maintainer>
+  <maintainer email="miguel.prada@tecnalia.com">Miguel Prada Sarasola</maintainer>
+  <maintainer email="nhg@ipa.fhg.de">Nadia Hammoudeh Garcia</maintainer>
+
+  <license>BSD</license>
+
+  <url type="website">http://moveit.ros.org/</url>
+  <url type="bugtracker">https://github.com/ros-planning/moveit_setup_assistant/issues</url>
+  <url type="repository">https://github.com/ros-planning/moveit_setup_assistant</url>
+
+  <author>Felix Messmer</author>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <!-- TODO(andyz): Re-enable when MoveIt2 debians are available -->
```

Comment:
is this because of CI? Otherwise using `.repos` file could help to build Moveit. On public projects, this should work fine.

Reply:
I am not sure why this is here. I hold it as I found it like that for remainder.